src/Microsoft.Identity.Client/Cache/Keys/MsalAccessTokenCacheKey.cs

```diff
@@ -35,7 +35,7 @@ namespace Microsoft.Identity.Client.Cache.Keys
     ///     format of the key is not important for this library, as long as it is unique.
     /// </summary>
     /// <remarks>The format of the key is platform dependent</remarks>
-    internal class MsalAccessTokenCacheKey
+    internal class MsalAccessTokenCacheKey : IiOSKey
```

Comment:
>IiOSKey [](start = 45, length = 7)

this seems odd to me. Why would a generic concept inherit from something platform specific?

Reply:
I will look at this in a separate PR. It is the responsability of token cache accessors (which are platform specific) to format the keys. I'd rather not introduce another factory etc. because the cache is isolated from all the factories.

Reply:
ok.